apps/cowswap-frontend/src/utils/currencyAmountToTokenAmount.ts

```diff
@@ -0,0 +1,11 @@
+import { Currency, CurrencyAmount, Token } from '@uniswap/sdk-core'
+
+import { Nullish } from 'types'
+
+export function currencyAmountToTokenAmount(amount: Nullish<CurrencyAmount<Currency>>): CurrencyAmount<Token> | null {
```

Comment:
nice. Very likely this method is useful in a bunch of orher places where we did it by hand over and over. We might want to follow up in another PR and find where else to use it

Reply:
Added in my TODO list